src/TheAlgorithms.jl

```diff
@@ -19,6 +19,7 @@ export line_length
 export SIR # TODO: make the name lowercase if possible
 export trapazoidal_area
 export is_armstrong
+export abs_val, abs_max, abs_min
```

Comment:
This is a bit of me being annoying, but in the prevision of this repository growing huge... I would prefer one function per line in lexicographical order. So, that is possible for reader/dev/maintainer to find out what is available/missing in just a glance.